### DISCUSSION TITLE
Why is sedimentation_impact_factor set to 0.5?

### DISCUSSION THREAD

**Discussion by depion** [contrailcirrus/pycontrails#103] [Q&A] [answered]
The file cocip_params.py contains the following default setting for `sedimentation_impact_factor`:

```
    #: Sedimentation impact factor. Denoted by :math:`f_{T}` in eq. (35) of
    #: :cite:`schumannContrailCirrusPrediction2012`.
    #: Schumann describes this as "an important adjustable parameter", and sets
    #: it to 0.1 in the original publication.
    sedimentation_impact_factor: float = 0.5
```

https://github.com/contrailcirrus/pycontrails/blob/d574d9d28367e847fa3565e80de35c2b369446b6/pycontrails/models/cocip/cocip_params.py#L153

Why is this set to 0.5 instead? Is this based on another study? If yes, this might be worth to be mentioned...?

**Comment by zebengberg** [comment, marked as answer]
Thanks for pointing this out @depion !

I think this value comes from paragraph 27 of [Schumann and Graf 2013](https://doi.org/10.1002/jgrd.50184).

@roger-teoh , can you confirm?

**Reply by zebengberg** [reply]
@roger-teoh is able to confirm this. He also says the default value of 0.5 agrees with the original fortran code he referenced.

I updated the docstring in a2fa351594106d92fa9286d787a7e16d3517034a
